### ISSUE TITLE
white-space style doesn't work

### ISSUE THREAD

**Issue by jagadyudha** [danomatic/react-pdf-html#47]
style `white-space: pre-wrap` doesn't work

This is what i want:
![image](https://user-images.githubusercontent.com/41937681/211443154-a0f34532-14df-4915-88ed-af7cb9830247.png)

But the spaces are gone when it is converted to a pdf:
![image](https://user-images.githubusercontent.com/41937681/211443328-10ba9f01-6bb6-4bcc-9ccb-eccba7b75b2c.png)

**Comment by danomatic** [comment]
Good catch. White space support is limited right now, with some work done to support the `pre` tag. The next step would be to do a more thorough dive into the different style values and make sure that each one has been implemented.

**Comment by JorgeAtPaladin** [comment]
Hey @danomatic , we'd love to either have white-space: pre or being able to have elements such as <pre><span>...</span></pre> work. Right now <pre> is wrongly implemented to render inner elements as raw text.
